perception/navigator_lidar_oa/src/ConnectedComponents.h

```diff
@@ -83,6 +83,9 @@ struct objectMessage
 	uint32_t pclInliers = 0;
 	geometry_msgs::Vector3 normal;
 	float minHeightFromLidar,maxHeightFromLidar;
+	bool current = false;
+	bool locked = false;
```

Comment:
One thing I would like is to be able to interact with these members from ros messages / the database service besides just rviz. So perhaps we should just add locked and real to the PerceptionObject message, and some commands to the database service for setting these/inserting new ones. This way we could write another program which, for example, inserts GPS waypoints with given object types, or presets some default "fake" object locations.